eng/AzSdk.reference.targets

```diff
@@ -1,17 +0,0 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="[3.3.18, 4.0.0)" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="[2.3.18, 3.0.0)" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(IsTargetingNetStandard)' == 'true'">
-    <PackageReference Include="System.Net.Http" Version="4.3.0"/>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
I think the data-plane builds are failing because of this removal. In particular I believe they need this Newtonsoft.Json reference.